packages/sdk/src/auth/user.ts

```diff
@@ -37,12 +41,15 @@ export async function getUserBySupabaseCookie(
       supabaseServerToken,
     )
     : { supabase: supabaseServerToken };
-  const { data: _user } = await supabase.auth.getUser(
-    accessToken,
-  );
+  const [{ data: _user }, jwt] = await Promise.all([
+    supabase.auth.getUser(
+      accessToken,
+    ),
+    jwtIssuer.verify(sessionToken),
+  ]);
   const user = _user?.user;
   if (!user) {
-    return undefined;
+    return jwt;
```

Comment:
What happens if jwtIssuer.verify fails? I mean, the sessionTOken isn't verified.

Reply:
returns undefined, so same as prev behavior